databricks/sdk/runtime/__init__.py

```diff
@@ -6,19 +6,42 @@ def _get_global_dbutils():
         return None
 
 
-dbutils = _get_global_dbutils()
+is_oss_implementation = True
```

Comment:
i think a better name could be “local implementation”

Reply:
Makes sense, let me change the name.